tests/test_conversationhandler.py

```diff
@@ -63,8 +64,12 @@ def reset(self):
                 CommandHandler('gettingThirsty', self.start),
                 CommandHandler('drinkMore', self.drink)
             ],
+            ConversationHandler.TIMEOUT: [
+                MessageHandler(None, self.passout)
```

Comment:
Please use a list of handlers here and add a test to check if both cvan be reached